.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -507,6 +507,25 @@ echo -e "\nCompleted installation of the Submitty homework submission server\n"
 
 ################################################################################################################
 ################################################################################################################
+# INSTALL PYTHON SUBMITTY UTILS
+
+echo -e "Intall python_submitty_utils"
+
+mkdir -p ${SUBMITTY_INSTALL_DIR}/python_submitty_utils
+chown root:root ${SUBMITTY_INSTALL_DIR}/bin
+chmod 700 ${SUBMITTY_INSTALL_DIR}/bin
+
+rsync -rtz ${SUBMITTY_REPOSITORY}/python_submitty_utils ${SUBMITTY_INSTALL_DIR}/python_submitty_utils
```

Comment:
When I manually copied these commands to test a branch I'm working on, I had to add a / to the end of the source directory (i.e. `${SUBMITTY_REPOSITORY}/python_submitty_utils/`) , otherwise I ended up with /usr/local/submitty/python_submitty_utils/python_submitty_utils